### ISSUE TITLE
Increase iframe height on mobile screens

### ISSUE THREAD

**Issue by jasomdotnet** [englishextra/iframe-lightbox#11]
Hi,
more then issue, this is ask for an advice. On mobile device, proportions of iframe are kept. How may I increase height of iframe (on mobile)? See [picture](https://i.imgur.com/kKYDsRv.png).

I have already fix width for mobiles:

```
@media (max-width: 767px) {
    .iframe-lightbox .content-holder {
        width: 95%;
    }
}
```
Thank you.

**Comment by englishextra** [comment]
@jasomdotnet I will see that. Thanks,

**Comment by englishextra** [comment]
@jasomdotnet I'm afraid 95% is making overflow on Desktop browsers. As for mobile it actually needs a very huge work to test all devices and to make appropriate sizes

**Comment by jasomdotnet** [comment]
95% is good* for width. I agree, that 95% is too much for height. Some slightly higher value would be fine: 75%? 80%? Now there is too much unused space on mobile (vertical) screen.

Also, better approach, IMHO, is to set (top and bottom) padding then specifying height (and width). Also because mobile can have vertical or horizontal position of the screen.

*star_note: I tested it on single device (my Samsung Android phone)

**Comment by englishextra** [comment]
The proplem with this lightbox and its feature that it makes an Aspect ratio container 

That's why no height is present in `.iframe-lightbox .content-holder ` and this is the difference from other lightboxes. The dev set their custom Aspect Ratio via `data-padding-bottom` and taking this to account and optimal 80% width is set to `.iframe-lightbox .content-holder `

In general lightbox as image magnifier/view increaser is useless because most images streched to full viewport width. Lightbox is needed on mobile when you have small thumbs on some internet shop site. 

So strategically I'm not sure if I have to ovedirect the dev's intentions to tweak the default CSS.

TL:DR I'm not sure if I really have to add some media query fixes or some padding tweaks. Because I can't predict what the dev is going to do exactly with the lightbox.

**Comment by jasomdotnet** [comment]
> I'm not sure if I really have to add some media query fixes or some padding tweaks. Because I can't predict what the dev is going to do exactly with the lightbox.

ok, it have sense. I think I understand your intend.

Several (subjective) points:
* I vote for padding tweaks, because it best fits my needs which are to iframe html page, not video with fixed dimensions (16:9)
* What about: if user don't define `data-padding-bottom` then padding tweaks will be present.
* consider to create pure 'iframe-html-page-lightbox' with padding tweaks and no fixed radio container :D :D

**Comment by englishextra** [comment]
0.1.9 on NPM is published.

> What about: if user don't define data-padding-bottom then padding tweaks will be present.

I guess this is where it could be interesting to move on

Thanks

**Comment by jasomdotnet** [comment]
what about this issue? is it still active?

**Comment by englishextra** [comment]
I need some time, because the priority is JS logic which should be resolved first then I'm thinking of paddings which is not that simple because of the Aspect Ratio.

**Comment by jasomdotnet** [comment]
what means 'wontfix'?

**Comment by englishextra** [comment]
@jasomdotnet Hi

This means that I see no use case. The goal of the lib is:

 - to keep the codebase as little as possible
 - to provide Aspect Ratio tuning opportunity

On mobile when you hold the screen vertically - the lightbox is almost useless, while turning the device horizontally makes the lib useful.

So to me it is not clear why to increase the iframe/lightbox height - that way the concept of AR is ruined.

Maybe you meant that on mobile the lightbox should forget about keeping the AR, then how about the Youtube video which in vertical position can't be of full height. 

So I don't have enough visual examples of what are you trying to achieve. But everything is possible.

**Comment by jasomdotnet** [comment]
Sound logic. I will close this issue for now. You did a good job with iframe-lightbox, so far. Thanks
